guardrails/cli/configure.py

```diff
@@ -4,75 +4,89 @@
 from os.path import expanduser
 from typing import Optional
 
+from guardrails.classes.credentials import Credentials
+from guardrails.cli.server.hub_client import get_auth
 import typer
 
 from guardrails.cli.guardrails import guardrails
 from guardrails.cli.logger import LEVELS, logger
-from guardrails.cli.server.hub_client import AuthenticationError, get_auth
 
 
-def save_configuration_file(token: str, no_metrics: bool) -> None:
+DEFAULT_TOKEN = ""
+DEFAULT_ENABLE_METRICS = True
+
+
+def save_configuration_file(
+    token: Optional[str], enable_metrics: Optional[bool]
+) -> None:
+    if token is None:
+        token = DEFAULT_TOKEN
+    if enable_metrics is None:
+        enable_metrics = DEFAULT_ENABLE_METRICS
+
     home = expanduser("~")
     guardrails_rc = os.path.join(home, ".guardrailsrc")
     with open(guardrails_rc, "w") as rc_file:
         lines = [
             f"id={str(uuid.uuid4())}{os.linesep}",
             f"token={token}{os.linesep}",
-            f"no_metrics={str(no_metrics).lower()}",
+            f"enable_metrics={str(enable_metrics).lower()}",
         ]
         rc_file.writelines(lines)
         rc_file.close()
 
 
+def _get_default_token() -> Optional[str]:
+    """Get the default token from the configuration file."""
+    return Credentials.from_rc_file(logger).token
+
+
 @guardrails.command()
 def configure(
     token: Optional[str] = typer.Option(
-        help="Your Guardrails Hub auth token.", hide_input=True, default=""
+        default_factory=_get_default_token,
+        help="Your Guardrails Hub auth token.",
+        hide_input=True,
+        prompt="Token (optional)",
+    ),
+    enable_metrics: Optional[bool] = typer.Option(
+        DEFAULT_ENABLE_METRICS,
+        "--enable-metrics/--disable-metrics",
```

Comment:
Does this actually work?  I know they have `--opt` and `--no-opt` automagically but couldn't find reference to this in the docs

Reply:
nvm just found it: https://typer.tiangolo.com/tutorial/parameter-types/bool/#alternative-names

Reply:
https://typer.tiangolo.com/tutorial/parameter-types/bool/#alternative-names